command/config/config.go

```diff
@@ -288,6 +288,7 @@ type EnvConfig struct {
 		MaxPoolSize          int    `envconfig:"DRONE_MAX_POOL_SIZE" default:"2"`
 		EnableAutoPool       bool   `envconfig:"DRONE_ENABLE_AUTO_POOL" default:"false"`
 		HarnessTestBinaryURI string `envconfig:"DRONE_HARNESS_TEST_BINARY_URI"`
+		PluginBinaryURI      string `envconfig:"DRONE_PLUGIN_BINARY_URI" default:"https://github.com/drone/plugin/releases/download/v0.1.2-alpha"`
```

Comment:
@shubham149 what do you think about linking to the latest binary by default for now?  If we can keep the command line flags stable, it means we can iterate without having to perform new lite engine releases?

> To link directly to a download of your latest release asset that was manually uploaded, link to /owner/name/releases/latest/download/asset-name.zip.

Reply:
There are a lot of possibilities for introducing a regression in anything that is using plugin binary if we use latest. Using a versioned binary reduces those issues. We will have automated tests that run as part of new releases in harness ci which will catch regressions. But that won't be possible with latest version usage.

We are following the same convention for lite-engine in runner and it has worked well for us.

Reply:
@shubham149 in Drone many of our customers use the `:latest` version of plugin.  It requires discipline to avoid breaking changes, but it means customers always get the latest and greatest.  It has worked well for Drone over the years.  It is ok if we are not ready to pull the `latest` release, but let's make this a goal for the future. Discipline and backward compatibility is a huge reason Drone sold over the years;  enterprises value stability over almost everything else.

Reply:
We can use versioned approach for now. Right now, we have a weekly release cadence for harness CI which should be good enough for most of the cases. 

I am worried about engineering team spending more time debugging the customer issues related to plugin in case we use latest version. We will need to add tests covering all the cases in plugin repo drone builds before we can move to latest release. Also, we will have to setup mechanisms for debugging customer issues with latest versioned approach.

Reply:
To add to Brads point, will the default value ever be used ? 
if it is always used  then there will need to be regular weekly releases of this runner. Which means that any new large changes going in, have to be introduced in a week. 
If this value is always overridden then using a default value of latest seems to be natural.